packages/lambda-tiler/src/cli/serve.ts

```diff
@@ -110,7 +110,7 @@ async function useLocal(): Promise<void> {
         const requestId = ulid.ulid();
         const logger = LogConfig.get().child({ id: requestId });
 
-        const tileSets = await Promise.all([...TileSets.cache.values()]);
+        const tileSets = await Promise.all([...TileSets.cache.values()].map((c) => c.value));
```

Comment:
Is this change for Cloning the TileSets values?

Reply:
this change is because we now store the time of when we fetched the object last.

so the `[....cache.values()]`  has gone form `TileSet[]` to
```typescript
{ value: TileSet, time: number}[]
```